### PR TITLE
Add FAQ links

### LINKED ISSUE #37: Hook up FAQ links across the app
There are several spots throughout the app that link off to the internal FAQ for support.

Let's use this description to identify 1) the locations in the app that need to link to the FAQ and 2) the exact URL to link to
- [x] **Welcome**
Design: https://zpl.io/VDzrg4v
Link Text: Learn more
Link URL: https://lockbox.firefox.com/faq.html#how-do-i-get-my-saved-logins-into-firefox-lockbox

- [ ] **You're all Set** (onboarding confirmation)
Design: https://zpl.io/V0yxoJQ
Link Text: 256-bit encryption
Link URL: https://lockbox.firefox.com/faq.html#what-security-technologies-does-firefox-lockbox-use 

- [x] **No Entries Found**
Design: https://zpl.io/bAqXRlB
Link Text: Learn more
Link URL: https://lockbox.firefox.com/faq.html#how-do-i-enable-sync-on-firefox

- [x] **No Matching Entries** (when searching)
Design: https://zpl.io/2j4XN1q
Link Text: Learn more
Link URL: https://lockbox.firefox.com/faq.html#how-do-i-create-new-entries

- [x] **Entry Detail**
Design: https://zpl.io/2EAZQyl
Link Text: Learn how to edit this entry
Link URL: https://lockbox.firefox.com/faq.html#how-do-i-edit-existing-entries

- [x] **Send Usage Data**
Design: https://zpl.io/awYA3LK
Link Test: Learn more
Link URL: https://lockbox.firefox.com/privacy.html

blocked by #73

### PR DESCRIPTION
**Pull request by devinreams**
Fixes #37

## Testing and Review Notes

Ideally, all of the FAQ pages will slide up from the bottom -- please let me know if you find a case where that's not happening. Also, I cut a corner with the security text from the onboarding confirmation screen - all of the text is a link, not just the `256-bit encryption` part, but hopefully that won't be that noticable.

## Work Remaining

- [x] add click on Welcome text
  - [x] set proper URL `#how-do-i-get-my-saved-logins-into-firefox-lockbox`
- [x] add click on "You're All Set" onboarding
  - [x] set proper URL `#what-security-technologies-does-firefox-lockbox-use`
  - [x] add action to nav graph
- [x] add click on No Entries Found button
  - [x] fix "null" crash unable to access this?
  - [x] set proper URL `* #how-do-i-enable-sync-on-firefox`
- [x] add click on No Matching button
  - [x] fix "null" crash unable to access this?
  - [x] set proper URL `#how-do-i-create-new-entries`
- [x] add click on Entry Detail
  - [x] set proper URL `#how-do-i-edit-existing-entries`
  - [x] add to navigation graph/map
- [x] add click on Send Usage Data
  - [x] set proper URL
  - [x] add action to nav graph

## To Do

- [x] add “WIP” to the PR title if pushing up but not complete nor ready for review
- [x] double check the original issue to confirm it is fully satisfied
- [x] add testing notes and screenshots in PR description to help guide reviewers
- [x] add unit tests
  - optional: consider adding instrumentation (integration/UI) tests
- consider running this branch in a debug simulator and check for memory leak notifications or any warnings
- [x] request the "UX" team perform a design review (if/when applicable)
- [x] make sure CI builds are passing (e.g.: fix lint and other errors)
- [x] check on the [accessibility](https://mozilla-lockbox.github.io/lockbox-android/accessibility/) of any added UI

### CODE REVIEW

File: app/src/main/java/mozilla/lockbox/adapter/ItemListAdapter.kt [hunk 1]
```diff
@@ -42,21 +45,30 @@ class ItemListAdapter : RecyclerView.Adapter<ItemListCell>() {
             NO_MATCHING_ENTRIES_CELL_TYPE -> {
                 val view = inflater.inflate(R.layout.list_cell_no_matching, parent, false)
 
+                view.noMatchingLearnMore
+                    .clicks()
+                    .subscribe(noMatchingEntriesClicks as PublishSubject)
```
Comment (eliserichards) [inline app/src/main/java/mozilla/lockbox/adapter/ItemListAdapter.kt]: Why is `noMatchingEntriesClicks` using `as PublishSubject`, but ln 59 and ln 71 are using `as Subject`? Can we make them consistent?

Comment (sashei) [reply app/src/main/java/mozilla/lockbox/adapter/ItemListAdapter.kt]: yesssssssssssssssssssssssssss

File: app/src/main/java/mozilla/lockbox/support/Constant.kt [hunk 2]
```diff
@@ -27,7 +27,17 @@ object Constant {
     }
 
     object Faq {
-        const val uri = "https://lockbox.firefox.com/faq.html#top"
+        const val uri = "https://lockbox.firefox.com/faq.html"
```
Comment (jhugman) [inline app/src/main/java/mozilla/lockbox/support/Constant.kt]: Who localizes this? i.e. should this come from `resources`?

File: app/src/main/res/navigation/graph_main.xml [hunk 3]
```diff
@@ -110,13 +124,28 @@
...
             android:label="@string/nav_menu_settings"
-            tools:layout="@layout/fragment_setting"/>
+            tools:layout="@layout/fragment_setting">
+        <action
+                android:id="@+id/action_setting_to_webview"
+                app:destination="@id/fragment_webview"
+                app:enterAnim="@anim/slide_in_bottom"
+                app:exitAnim="@anim/none"
+                app:popExitAnim="@anim/slide_out_bottom"
+                app:popEnterAnim="@anim/none"/>
```
Comment (jhugman) [inline app/src/main/res/navigation/graph_main.xml]: All these actions are going to the same place, with the same animations. Might they be better as global actions?

File: app/src/main/java/mozilla/lockbox/support/Constant.kt (hunk 2 above)
Comment (sashei) [reply app/src/main/java/mozilla/lockbox/support/Constant.kt]: Good question; I have no idea what the l10n story is for our FAQ pages. maybe @devinreams knows more?

Comment (devinreams) [reply app/src/main/java/mozilla/lockbox/support/Constant.kt]: Thanks for bringing it up. Long term the pages/content will move to SUMO and will add this to our localization infrastructure todo list/questions.

For now, and I think you all agree, I don't think we need to try to future-proof or handle it quite yet...

**Comment by devinreams** [comment]
Thank you for the help and work getting this finished up @sashei, and for everyone helping review it! ❤️

**Comment by devinreams** [comment]
Re-tested all of this myself on Pixel simulator (Version 8) after `master` merged and everything is still working A-OK. Merging! 🚢

**Pull request merged**